inst/pages/alpha_diversity.qmd

```diff
@@ -1,98 +1,115 @@
-
-**Richness** refers to the total number of species in a community
-  (sample). The simplest richness index is the number of species
-  observed in a sample (observed richness). Assuming limited sampling
-  from the community, however, this may underestimate the true species
-  richness. Several estimators have been developed to address this,
-  including for instance ACE [@Chao1992] and Chao1 [@Chao1984]
-  indices. Richness estimates do not aim to characterize variations in
-  species abundances.
+## Background
+
+Alpha diversity, or within-sample diversity, is a central concept in microbiome 
+research. In ecological literature, several distinct but related alpha diversity
+indices, often referring to **richness** and **evenness** - the number of taxa 
+and how they are distributed, respectively - are commonly used 
+[@Willis2019;@Whittaker1960]. The term **diversity** can be used to collectively 
+refer to all these indices.
+
+### Applications
+
+Alpha diversity is predominantly used to quantify complexity in the microbiome. 
+In the general adult population, lower alpha diversity and lower bacterial load 
+have been associated to worse overall physical and mental health 
+[@Valles-Colomer2019GBMs;@vandeputte2017quantitative]. However, this principle 
+may not generalize to other populations, most notably in early life
+and in patient cohorts [@Ma2019].
+
+### Approaches
+
+The majority of alpha diversity metrics are closely related, though this is not 
+evident from their names. Bastiaanssen et al. [-@bastiaanssen2023bugs1] lay out
+this relationship across two factors (See table below); First, alpha diversity 
+metrics can be defined as special cases of a unifying equation of **diversity**, 
+where the **Hill number** determines the specific index captured. Lower Hill 
+numbers favour **richness**, the number of distinct taxa, whereas higher numbers 
+favour **evenness**, how the taxa are distributed over the sample [@Hill1973]. 
+Second, some alpha diversity metrics are weighed based on phylogeny, like 
+Faith's PD [-@Faith1992] and PhILR [@Silverman2017]. 
+
+```{r, echo=FALSE}
+library(gt)
+
+#Descriptions
+neutral_div <- 
+  "**Neutral Diversity**  
+
+---  
+*Weighs each taxon equally, no assumptions about phylogeny*"
+phyl_div    <- 
+  "**Phylogenetic Diversity**  
+
+---  
+*Indices are scaled based on taxonomic closeness with a phylogenetic tree*"
+
+hill_0 <- 'Dependent on presence and absence of taxa, not abundance'
+hill_1 <- 'Dependent on how evenly taxa are distributed in a sample'
+hill_2 <- 'The probability of two randomly picked taxa not being the same'
+
+table_colnames <- c("phyl_desc", hill_0, hill_1, hill_2)
+
+footnote_md <- md(
+"The equation for general diversity can be defined as follows: 
+   \\${{}^qD = (\\sum_{i=1}^{R}p^q_i})^{\\frac{1}{(1-q)}}\\$,   
+   with *q* = Hill number, *R* = number of features, 
+   *p* = relative feature abundance.")
+
```

Comment:
For some reason, this does not work for me. I get this error:

Error: ParseError: KaTeX parse error: Unexpected character: '\' at position 50: …frac{1}{(1-q)}}\̲